DQM/SiStripMonitorClient/python/customizeForNoTrackerDCS.py

```diff
@@ -0,0 +1,35 @@
+"""
+Module to remove SiStrip DCS checks in Strip and Tracking Monitors
+"""
+
+import FWCore.ParameterSet.Config as cms
+
+def producers_by_type(process, *types):
+    return [module for module in process._Process__producers.values() if module._TypedParameterizable__type in types]
+
+def removeDCSChecks(process, acceptedParts):
+    print('WARNING: removing SiStrip DCS Checks in Strip and Tracking Monitors')
+
+    for producerType in ['SiStripMonitorTrack', 'SiStripMonitorCluster']:
+        for producer in producers_by_type(process, producerType):
+            producer.UseDCSFiltering = cms.bool(False)
+                    
+    for producer in producers_by_type(process, 'SiStripMonitorCluster'):
+        producer.StripDCSfilter.dcsPartitions = cms.vint32(acceptedParts)
+
+    for producer in producers_by_type(process, 'TrackingMonitor'):
+        producer.genericTriggerEventPSet.dcsPartitions = cms.vint32(acceptedParts)
+
+    return process
+
+def removeStripDCSChecks(process):
```

Comment:
It's a bit confusing to have a function that is `removeXYZ` but then in the options you specify specific ones to KEEP, could we make it explicit like

```
removeDCSChecks(process, acceptedParts=[28,29]) 
```

and similar?

Reply:
isn't this exactly what's going on in 

https://github.com/cms-sw/cmssw/blob/8560e6788658e89579dff83873fcf64ef5e3f052/DQM/SiStripMonitorClient/python/customizeForNoTrackerDCS.py#L26

https://github.com/cms-sw/cmssw/blob/8560e6788658e89579dff83873fcf64ef5e3f052/DQM/SiStripMonitorClient/python/customizeForNoTrackerDCS.py#L30

https://github.com/cms-sw/cmssw/blob/8560e6788658e89579dff83873fcf64ef5e3f052/DQM/SiStripMonitorClient/python/customizeForNoTrackerDCS.py#L34